_posts/2018-11-30-the-year-of-k.md

```diff
@@ -0,0 +1,22 @@
+---
+layout:     post
+title:      The Year of K at Sky Betting & Gaming
+date:       2018-11-30
+summary:    Kotlin, Kubernetes and Kafka - why these Ks are our onwards tech choices.
+category:   Technologies
+tags:       kotlin,kubernetes,k8s,kafka,technologies,techniques
+author:     kevin_bowman
+---
+Kotlin, Kubernetes, and Kafka; the newest kids on the block are all "K" themed technologies.  This trio aren’t exactly "new", though they are definitely on the more recent end of the spectrum.  At Sky Betting & Gaming we use a variety of technologies - we're far from being a monoculture, our approach is to allow each "tribe" to develop its own technical strategy within sensible principles - and We use them predominantly in our "Nexus" tribe, which is building some of our fundamental future strategic platforms, and uniquely has the luxury of being able to make greenfield choices.  With this freedom, we put some of our best technical minds together to architect the most maintainable, sustainable, and operatable setup possible.
```

Comment:
This has a very long sentence, which is quite difficult to read.

Reply:
Maybe link to the product websites on the first mentions of Kotlin, Kubernetes and Kafka.